Create branch `V3.0 dev` for mmeta V3.0 and initial commit

esuxiao opened the pull request:
Create new branch for development of `mmeta V3.0`
Refer to issues:

- #9

esuxiao committed:
Add ggplot2 package as depends.

esuxiao committed:
Update export field and remove S3 registration

esuxiao committed:
Remove deprecated *.Rd file

esuxiao committed:
update R functions which are release to dev

esuxiao committed:
Change version from 2.4 to 3.0

esuxiao commented on R/MultipleTables-method-exported.R:
```diff
@@ -0,0 +1,165 @@
+  
+  ## print out report if user requests
+  if(verbose){
+    MultipleTables._summaryPrint(multiple_tables_object)
```
Only when verbose == True, the MultipleTables._summaryPrint(multiple_tables_object) function will be called then the cat functions within MultipleTables._summaryPrint() can be called.

jchen1992 committed:
Update DESCRIPTION

jchen1992 committed:
Update DESCRIPTION

jchen1992 reviewed (changes requested):
ggplot2 should be added to imports not depends.

jchen1992 committed:
ggplot2 was removed from depends to imports

usually, we list the version of R required in the Depends field and the package in the Imports field.

jchen12340529 committed:
Update DESCRIPTION

jchen12340529 committed:
Create SingleTable.create.Rd

jchen12340529 committed:
Update SingleTable-methods-exported.R

jchen12340529 committed:
Update colorectal.Rd

jchen12340529 committed:
Update diabetes.Rd

jchen12340529 committed:
Update SingleTable.create.Rd

jchen12340529 committed:
Update withdrawal.Rd

jchen12340529 committed:
Update SingleTable-methods-exported.R

esuxiao committed:
Readme: add method to install from Github

esuxiao committed:
Add StagedInstall: hard-coded installation error

esuxiao committed:
readme: add installation method

esuxiao commented on DESCRIPTION:
```diff
@@ -14,15 +14,17 @@ Authors@R: c(person("Sheng", "Luo", role="aut"),
 Language: en-US
+Encoding: UTF-8
+RoxygenNote: 7.2.2
+StagedInstall: no 
```
Try to solve ERROR: hard-coded installation path: please report to the package maintainer and use '--no-staged-install' problem. Refer to https://community.rstudio.com/t/no-staged-install-error/35321/2

jchen12340529 committed:
Update DESCRIPTION

jchen12340529 committed:
Create SingleTable.modelFit.Rd

jchen12340529 committed:
Update SingleTable-methods-exported.R

jchen12340529 committed:
Update SingleTable-methods-exported.R

Add comments to update Rd file by Roxygen2.

jchen12340529 committed:
Update SingleTable.create.Rd

jchen12340529 committed:
Update SingleTable.create.Rd

jchen12340529 committed:
Update SingleTable.modelFit.Rd

jchen12340529 committed:
Create SingleTable.summary.Rd

jchen12340529 committed:
Update SingleTable-methods-exported.R

jchen12340529 committed:
Update the SingleTable.summary Rd file

jchen12340529 committed:
Update SingleTable-methods-exported.R

jchen12340529 committed:
Update singletable.plot Rd files

jchen12340529 committed:
Corrected the return value of singleTable.plot

return value:
None -> a ggplot2 object is returned.

jchen12340529 committed:
Update MultipleTables-method-exported.R

jchen12340529 committed:
Update singletable rd files

jchen12340529 committed:
Create MultipleTables.create.Rd

jchen12340529 committed:
Create MultipleTables.modelFit.Rd

jchen12340529 committed:
Create MultipleTables.plot.Rd

jchen12340529 committed:
Create MultipleTables.summary.Rd

jchen12340529 committed:
minor correction for R markdown

 95% credible/confidence intervals -》95\% credible/confidence intervals are plotted.

jchen12340529 committed:
Bug fixing due to R package check

This upload including several minor issues raised by R check.

esuxiao committed:
ignore *.o files which are compiled objects

jchen1992 committed:
Upload Rcheck related to the current v3.0 dev

esuxiao committed:
Delete Rcheck folder

esuxiao committed:
delete .o file which should be excluded from source

jchen1992 committed:
checklog for 3.0 dev

esuxiao committed:
Delete check log file

esuxiao committed:
exclude *.log file in the future

jchen12340529 committed:
Fixed the error in check.log

Error has been corrected by adding the following codes to the examples of Multitables.plot()

## Summary of the fitting process (default)
 multiple_tables_obj_exact <- MultipleTables.summary(multiple_tables_obj_exact)

jchen12340529 committed:
Remove parmsPosteriorIndependentCal

replace parmsPosteriorIndependentCal with parmsPosterior;

Replace parms_prior to parms

jchen12340529 committed:
fixed study_id, ET_lower, ET_upper

Add specific_summary$ before each variable.

jchen12340529 committed:
Add News file

jchen12340529 committed:
Version number need 3 components. (3.0.0))

jchen12340529 committed:
fixing minor issues

jchen1992 commented:
@esuxiao I have merged the V3.0_dev to the main branch. Do you want to delete the V3.0_dev branch?
